libs/core/src/schemas/index.ts

```diff
@@ -0,0 +1 @@
+export * from './community.schema';
```

Comment:
export * as community from './community.schema'; ... so we can later use schemas.community...?